<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TurntableControllerBase</class>
 <widget class="QMainWindow" name="TurntableControllerBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="enableMotor_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Motor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disableMotor_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable Motor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="movePlusOne_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+1 deg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveMinusOne_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-1 deg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Move By Fixed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Move By Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendCommand_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Command</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="moveAmount_spinBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="portName_lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COM4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatus_label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>22</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>ConnectionStatus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getPosition_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Position</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="degIndicator">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Degrees</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>460</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1/100ths Degrees</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="deg100Indicator">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>480</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="tryConnect_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Try Connect</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="incomingMessages_plainTxtEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closeConnection_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Close 
Connection</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
